CONTRIBUTING.md

```diff
@@ -149,29 +148,29 @@ This facilitates social contribution, easy testing, and peer review.
 To contribute changes, use the following workflow:
 
 1. [**Fork the repository**](https://github.com/ConsenSys/web3signer/fork).
-1. **Clone your fork** to your computer.
-1. **Create a topic branch** and name it appropriately.
+2. **Clone your fork** to your computer.
```

Comment:
Github/markdown will convert 1. 1. 1. into 1. 2. 3. automatically. Makes it easier to maintain I guess since you don't have to manually renumber.

Reply:
Makes sense. Reverted that that change. Thanks!

Reply:
Makes sense. Reverted that change. Thanks!